### DISCUSSION TITLE
Display multiple images in Lightbox

### DISCUSSION THREAD

**Discussion by Adstokoe19** [igordanchenko/yet-another-react-lightbox#97] [Q&A] [answered]
Is is possible to display a grid of images in the lightbox? I would like to display 4 images (16:9) when the lightbox is open. I could use a custom slide but was wondering if there is a pre-built solution for this?

Thanks

**Comment by igordanchenko** [comment, marked as answer]
There is currently no pre-built solution for this use case, and a custom slide sounds like a perfect approach to implement it.
